apps/src/applab/applab.js

```diff
@@ -697,22 +697,6 @@ Applab.init = function (config) {
   var onMount = function () {
     studioApp.init(config);
 
-    var viz = document.getElementById('visualization');
-    var vizCol = document.getElementById('visualizationColumn');
-
-    if (!config.noPadding) {
-      viz.className += " with_padding";
-      vizCol.className += " with_padding";
-    }
-
-    if (studioApp.reduxStore.getState().pageConstants.isEmbedView || config.hideSource) {
-      // no responsive styles active in embed or hideSource mode, so set sizes:
-      viz.style.width = Applab.appWidth + 'px';
-      viz.style.height = (shouldRenderFooter() ? Applab.appHeight : Applab.footerlessAppHeight) + 'px';
-      // Use offsetWidth of viz so we can include any possible border width:
-      vizCol.style.maxWidth = viz.offsetWidth + 'px';
-    }
-
```

Comment:
Side effect of this change. Previously our VisualizationOverlay would call `VisualizationOverlay.prototype.recalculateTransformAtScale_` before we had added the `with_padding` class. Adding this class results in a 1px border, which actually means our transform is off by 1px. By moving this into React, we already have the border when calling `recalculateTransformAtScale_` and we end up with the right transform (albeit 1 px different than before).